src/acquisition/covidcast/database.py

```diff
@@ -342,4 +350,48 @@ def retrieve_covidcast_meta_cache(self):
     cache_hash = {}
     for entry in cache:
       cache_hash[(entry['data_source'], entry['signal'], entry['time_type'], entry['geo_type'])] = entry
+    if not len(cache) and table_name=='covidcast_meta_cache_test': # TODO: remove line
+      print("test cache empty -- falling back to regular cache ; this should happen ONLY ONCE (after regular cache populated)") # TODO: remove line
```

Comment:
Don't need to do it now but please add a todo or issue for me to add log statements here.

Reply:
done